benchmarks/user/benchmark.py

```diff
@@ -433,12 +433,12 @@ def param_string(self, params):
 
 
 def get_ob_exec(func):
-    """Return a special ``opescibench.Executor`` to execute performance runs."""
+    """Return a special ``devitobench.Executor`` to execute performance runs."""
     try:
-        from opescibench import Executor
+        from devitobench import Executor
```

Comment:
Subfolder/module in devitobench is still called opescibench so this "may" break.

Reply:
Fixed.